js/interface.js

```diff
@@ -899,10 +899,10 @@ var DynamicLists = (function() {
                   id: index + 1,
                   columns: dataSourceColumns,
                   column: column,
-                  type: 'text',
+                  type: listLayout === 'agenda' ? 'html' : 'text',
```

Comment:
We would only want the content to be using the `html` field type. This change seems to risk all fields in the agenda layout to be added as a `html` type by default. That's too broad.